packages/perseus/src/__tests__/widgets.test.ts

```diff
@@ -1,5 +1,5 @@
 import {testDependencies} from "../../../../testing/test-dependencies";
-import {setDependencies} from "../dependencies";
+import * as dependencies from "../dependencies";
```

Comment:
**nit**: I think everywhere that we use `import *` we name the import using a capital. 🤷‍♂️  

```suggestion
import * as Dependencies from "../dependencies";
```